app/models/manageiq/providers/openstack/cloud_manager/vm/remote_console.rb

```diff
@@ -13,7 +13,7 @@ def remote_console_acquire_ticket(_console_type, _proxy_miq_server = nil)
       return nil, url, nil, nil, nil, 'novnc_url', nil
     end
 
-    def remote_console_acquire_ticket_queue(protocol, userid, proxy_miq_server = nil)
+    def remote_console_acquire_ticket_queue(protocol, userid)
```

Comment:
@Fryguy : yes, that's my understanding. See `app/controllers/api_controller/action.rb`